tests/functions_tests/test_clipped_relu.py

```diff
@@ -0,0 +1,62 @@
+import unittest
+
+import numpy
+
+import chainer
+from chainer import cuda
+from chainer import functions
+from chainer import gradient_check
+from chainer import testing
+from chainer.testing import attr
+
+if cuda.available:
+    cuda.init()
+
+
+class TestClippedReLU(unittest.TestCase):
+
+    def setUp(self):
+        self.x = numpy.random.uniform(-1, 1, (3, 2)).astype(numpy.float32)
+        self.gy = numpy.random.uniform(-1, 1, (3, 2)).astype(numpy.float32)
+        self.z = 0.75
+
+    def check_forward(self, x_data):
+        x = chainer.Variable(x_data)
+        y = functions.clipped_relu(x, self.z)
```

Comment:
Check the type of `y`. I mean,
```
self.assertEqual(y.data.dtype, numpy.float32)
```

Sometimes a result of a function becomes other types such as `numpy.float64` by mistake.